pymeasure/instruments/siglenttechnologies/siglent_spd1168x.py

```diff
@@ -38,21 +40,21 @@ def __init__(self, adapter, **kwargs):
         self.ch = {}
         self.ch[1] = SPDChannel(self, 1)
 
+    enable_4W_mode = Instrument.setting(
```

Comment:
I'm confused, why is this property now duplicated in the two child classes?

Reply:
Also, from the name (having a verb) this should be a method `enable_4W_mode(enabled):`